ui/admin/app/routes/scopes/scope/auth-methods/auth-method.js

```diff
@@ -22,6 +23,35 @@ export default class ScopesScopeAuthMethodsAuthMethodRoute extends Route {
 
   // =actions
 
+  /**
+   * Copies the contents of string array fields in order to force the instance
+   * into a dirty state.  This ensures that `model.rollbackAttributes()` reverts
+   * to the original expected array.
+   *
+   * The deep copy implemented here is required to ensure that both the
+   * array itself and its members are all new.
+   *
+   * @param {authMethodModel} authMethod
+   */
+  @action
+  edit(authMethod) {
```

Comment:
I think we could simplify this part and also delete the need for `ember-copy`.

We can use [structuredClone()](https://developer.mozilla.org/en-US/docs/Web/API/structuredClone) for this purpose.

I tested this and to me ir worked but maybe I am missing context on `ember-data` dirty state or something.

```javascript
edit(authMethod) {
  // Deep copy authMethod model
  authMethod = structuredClone(authMethod);
}
```

Reply:
That's really cool, I wasn't aware of that API. I would totally be down for using that and replacing `ember-copy`. It would be outside the scope of changing it in this PR but I think it's worth discussing; I'm not aware of any advantage of using `ember-copy`.

Reply:
Yeah, after last change isn't working as expected and I completly agree, this is out of the scope for this PR.

I will create a ticket on tech debt to explore deprecating `ember-copy` in favour of `structuredClone` or similar API's. I will also bring this topic to FE tech time to mention it 😃